app/views/admin/loans/timeline/_timeline_header.slim

```diff
@@ -1,33 +1,41 @@
-header.row.timeline-header
+header.timeline-header
   form accept-charset="UTF-8" method="post"
     input name="_method" type="hidden"
 
-    div.col-lg-2.col-md-2.col-sm-6.col-xs-6.first-col
-      div.btn-group
-        label.btn.btn-default
-          input type="checkbox" name="choose-all" id="choose-all"
-
-        button.btn.btn-default.dropdown-toggle type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
-          span.caret
-          span.sr-only = t(:dropdown_toggle)
-
-        ul.dropdown-menu
-          li
-            a id="check-all-ctrl" = t(:all)
-          li
-            a id="uncheck-all-ctrl" = t(:none)
-          li
-            a id="check-completed-ctrl" = t(:completed)
-          li
-            a id="check-incomplete-ctrl" = t(:incomplete)
-
-        input type="hidden" name="step-ids" id="step-ids"
-
-    div.col-lg-2.col-lg-2.col-md-2.col-sm-6.col-xs-6
-      div.btn-group
-        button.btn.btn-default.dropdown-toggle type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
-          = t("loan.batch_actions")
-          span.caret
-          span.sr-only = t(:dropdown_toggle)
-
-        = render "admin/loans/timeline/batch/batch_actions"
+    .row
+      .col-sm-4
+        .btn-group role="group" aria-label="Select"
+          label.btn.btn-default
+            input type="checkbox" name="choose-all" id="choose-all"
+
+          button.btn.btn-default.dropdown-toggle type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
+            span.caret
+            span.sr-only = t(:dropdown_toggle)
+
+          ul.dropdown-menu
+            li: a id="check-all-ctrl" = t(:all)
+            li: a id="uncheck-all-ctrl" = t(:none)
+            li: a id="check-completed-ctrl" = t(:completed)
+            li: a id="check-incomplete-ctrl" = t(:incomplete)
```

Comment:
Nice job refactoring here. Please make a point to discuss some slim coding tips with Liz (assuming she wrote this).